zipline/utils/paths.py

```diff
@@ -0,0 +1,223 @@
+"""
+Canonical path locations for zipline data.
+
+Paths are rooted at $ZIPLINE_ROOT if that environment variable is set.
+Otherwise default to expanduser(~/.zipline)
+"""
+from errno import EEXIST
+import os
+from os.path import exists, expanduser, join
+
+import pandas as pd
+
+
+def hidden(path):
+    """Check if a path is hidden.
```

Comment:
this is mainly to filter out the `.cache` dir we write. Until we really care about it being hidden I think this is fine